FRC1710-2019/src/main/java/frc/robot/Ballmech.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot;
```

Comment:
I tried removing `package` from the code and leaving just `frc.robot;`, but it gave me an error. I clicked on the recommended fix and it inserted `package frc.robot;` two lines above the new `frc.robot;`. `package frc.robot;` then gained an error on its semicolon, saying that an import should be what follows it. Unless anyone has a fix for this, I will probably just leave it as is, since it will already build in its current state.

Reply:
Ah ok, sorry I was confused, I thought it needed more than `package frc.robot` leave it as it if it works.

Reply:
It should look like that so don't worry.